src/main/java/core/basesyntax/handlers/OperationIncreaseHandler.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.handlers;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.exeptions.InvalidAmountException;
+import core.basesyntax.model.product.Fruit;
+import java.util.Optional;
+
+public class OperationIncreaseHandler implements OperationHandler {
+    private final StorageDao storageDao;
+
+    public OperationIncreaseHandler(StorageDao storageDao) {
+        this.storageDao = storageDao;
+    }
+
+    @Override
+    public int apply(Fruit fruit, int amount) {
+        Optional<Integer> amountFromStorage = storageDao.get(fruit).isEmpty()
+                ? Optional.of(0) : storageDao.get(fruit);
+        if (amount < 0) {
+            throw new InvalidAmountException();
+        }
```

Comment:
you replied that you have fixed the comment from Maxim, maybe you forgot to push changes?

Reply:
I have a default message in the class InvalidAmountException (cause it's a small exception that cover only one point)
Isn't it good?